src/amo/components/SearchResult/index.js

```diff
@@ -73,7 +85,10 @@ export class SearchResultBase extends React.Component<Props> {
     }
 
     let summary = null;
-    if (showSummary) {
+    // Because static themes are technically an extension type is has a summary
+    // field, but we want it to look like a theme which does not display this
+    // or description field here
+    if (showSummary && !isTheme) {
```

Comment:
I understand, but why are we doing this "here"? Can't we change the value of `showSummary` in the parent component?